staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -2418,7 +2418,7 @@ func (c *Cloud) waitUntilVolumeAvailable(volumeName KubernetesVolumeID) error {
 		return err
 	}
 
-	err = wait.Poll(encryptedCheckInterval, encryptedCheckTimeout, func() (done bool, err error) {
+	err = wait.Poll(createEBSCheckInterval, createEBSCheckTimeout, func() (done bool, err error) {
```

Comment:
I expected this to use exponential backoff and not a flat polling.

Reply:
Flat polling works pretty well so far. This change should focus on fixing the regression instead of adding new change. WDYT

Reply:
If upper limit as proposed in this PR is 30 seconds, we are probably fine. But, I am curious if 30s is indeed enough as upper bound of time limit for volume to become available. 

Even with 30s - we will currently poll approx. 10 times when checking for volume to become available. On average at least 2-3 times but if we used exponential backoff we will just check 5 times. From my experience - these numbers tend to add up and cause a cluster's API usage to go up. Since we have been trying to reduce AWS API usage as much as possible, my vote is to increase the max. wait time to higher value (may be 2mins. In recent benchmarks @jsafrane did he found 8s to be upper bound for 250 vols. he created) and use exponential backoff. 

We could also may be talk to EBS team about expected latency when volume becomes available after creation. Not waiting properly in `CreateVolume` call will cause created volume to become orphan.

Reply:
Please note - issues around this may have been masked until now because afaik most clusters use unencrypted volumes and hence we may not have sufficient evidence to prove that 30s time limit worked.

Reply:
committed exponential backoff for 2 minutes